pages/services/kafka/2.6.0-2.2.1/configuration/index.md

```diff
@@ -0,0 +1,84 @@
+title: Configuration
+menuWeight: 20
+model: /services/kafka/data.yml
+render: mustache
+---
+
+#include /services/include/configuration-install-with-options.tmpl
+#include /services/include/configuration-service-settings.tmpl
+#include /services/include/configuration-regions.tmpl
+
+## Configuring the ZooKeeper Connection
+
+{{ model.techName }} requires a running ZooKeeper ensemble to perform its own internal accounting. By default, the DC/OS {{ model.techName }} Service uses the ZooKeeper ensemble made available on the Mesos masters of a DC/OS cluster at `master.mesos:2181/dcos-service-<servicename>`. At install time, you can configure an alternate ZooKeeper for {{ model.techName }} to use. This enables you to increase {{ model.techName }}'s capacity and removes the DC/OS System ZooKeeper ensemble's involvement in running it.
+
+To configure an alternate Zookeeper instance:
+
+1. Create a file named `options.json` with the following contents. If you are using the [DC/OS Apache ZooKeeper service](/services/{{ model.kafka.zookeeperPackageName }}), use the DNS addresses provided by the `dcos {{ model.kafka.zookeeperPackageName }} endpoints clientport` command as the value of `kafka_zookeeper_uri`.
+
+    Here is an example `options.json` which points to a `{{ model.kafka.zookeeperPackageName }}` instance named `{{ model.kafka.zookeeperServiceName }}`:
+
+    ```json
+    {
+      "kafka": {
+        "kafka_zookeeper_uri": "zookeeper-0-server.{{ model.kafka.zookeeperServiceName }}.autoip.dcos.thisdcos.directory:1140,zookeeper-1-server.{{ model.kafka.zookeeperServiceName }}.autoip.dcos.thisdcos.directory:1140,zookeeper-2-server.{{ model.kafka.zookeeperServiceName }}.autoip.dcos.thisdcos.directory:1140"
+      }
+    }
+    ```
+
+1. Install {{ model.techName }} with the options file you created.
+
+    ```bash
+    dcos package install {{ model.packageName }} --options="options.json"
+    ```
+
+You can also update an already-running {{ model.techName }} instance from the DC/OS CLI, in case you need to migrate your ZooKeeper data elsewhere.
+
+<p class="message--note"><strong>NOTE: </strong> Before performing this configuration change, you must first copy the data from your current ZooKeeper ensemble to the new ZooKeeper ensemble. The new location must have the same data as the previous location during the migration.</p>
+
+```bash
+dcos {{ model.packageName }} --name={{ model.serviceName }} update start --options=options.json
+```
+
+## Extend the Kill Grace Period
+
+When performing a requested restart or replace of a running broker, the {{ model.techShortName }} service will wait a default of `30` seconds for a broker to exit, before killing the process. This grace period may be customized via the `brokers.kill_grace_period` setting. In this example we will use the DC/OS CLI to increase the grace period delay to `60` seconds. This example assumes that the {{ model.techShortName }} service instance is named `{{ model.serviceName }}`.
```

Comment:
Rewrite `In this example we will use the DC/OS CLI to increase the grace period delay to `60` seconds.` with `In this example, the DC/OS CLI is used to increase the grace period delay to `60` seconds.`